.gitignore

```diff
@@ -1,6 +1,6 @@
 *.pyc
 *.swp
-*.txt
+log.txt
```

Comment:
I think we can probably just remove .txt as a whole? Since I believe log's send with a .log extension

Reply:
oops yea i'm not sure I meant to include this. I hadn't realized we already had a requirements.txt, I had made one for local dev. I'll resolve